mediafile.py

```diff
@@ -1956,9 +1956,21 @@ def as_dict(self):
     )
     catalognums = ListMediaField(
         MP3ListDescStorageStyle('CATALOGNUMBER', split_v23=True),
+        MP3ListDescStorageStyle('CATALOGID', read_only=True),
+        MP3ListDescStorageStyle('DISCOGS_CATALOG', read_only=True),
         MP4ListStorageStyle('----:com.apple.iTunes:CATALOGNUMBER'),
+        MP4ListStorageStyle(
+            '----:com.apple.iTunes:CATALOGID', read_only=True
+        ),
+        MP4ListStorageStyle(
+            '----:com.apple.iTunes:DISCOGS_CATALOG', read_only=True
+        ),
         ListStorageStyle('CATALOGNUMBER'),
+        ListStorageStyle('CATALOGID', read_only=True),
+        ListStorageStyle('DISCOGS_CATALOG', read_only=True),
```

Comment:
Just checking whether we're being overzealous here… I understand that these strings might appear in the free-form tag formats, but do we actually know whether other software is putting analogous tags in stuff like MP4 frames? It seems best to be conservative and only include tags that we know are actually in use…

Reply:
mp3tag supports MP4, this is how an mp4 frame looks like when tagged with it:

```
----:com.apple.iTunes:CATALOGID=MP4FreeForm(b'M.PM18', <AtomDataType.UTF8: 1>)
```

I might have understood something wrong here: I thought `ListStorageStyle` is ment for FLAC and Vorbis files, analogous to `StorageStyle`. That not the case?